libs/elastic-indexing/src/lib/indexing.service.ts

```diff
@@ -43,4 +46,48 @@ export class IndexingService {
     logger.info('Done with sync')
     return true
   }
+
+  async rankEvaluation(locale: string) {
+    const requests = await this.getRankQueries(locale)
+    const index = await this.getIndex(locale)
+
+    return this.elasticService.precisionEvaluation(index, requests, 10)
```

Comment:
We might want to format the results to a more readable format for each query.
We should probably also document this endpoint

Reply:
I do agree to document this endpoint, that's an oversight on my part. I thought of this endpoint as an endpoint where developers can run the rank evaluation when they choose (during changes to the search engine) and also this could be hook some automation if needed/wanted.

But how would you like to see this formatted ?

I personally find json easy to decipher and with some basic browser extension, it's very easy to read
![image](https://user-images.githubusercontent.com/7864222/97345298-c8952f00-1881-11eb-95da-fda329188fbb.png)

Reply:
JSON is fine, I'm suggesting we format the response.
We have some information in this response that are not relevant e.g. meta.connection.resurrectTimeout
We might create a rank type e.g.
```ts
interface SearchRanking {
  score: number
  relevantDocsRetrived: number
  unratedDocsRetrived: number
  description: string <- what this ranking is testing
}
```
And return it in a ranking response e.g.
```ts
interface RankingResponse {
  [requestedString: string]: SearchRanking
}
```